src/tests/mocks/camera_mocks.cpp

```diff
@@ -8,6 +8,9 @@
 #include <components/camera/server.hpp>
 #include <tests/test_utils.hpp>
 
+namespace viam {
+namespace cppsdktests {
```

Comment:
This still has the problem of declaring functions like `fake_properties` in the namespace for all of the tests but other tests (like motor) will also have `fake_properties` (if it follows the same pattern). Thoughts?